doc/api/encoding.md

```diff
@@ -0,0 +1,63 @@
+# Encoding
+
+<!-- introduced_in=REPLACEME -->
+
+> Stability: 1 - Experimental
+
+<!-- source_link=lib/encoding.js -->
+
+The `node:encoding` module provides unicode validation and transcoding.
+To access it:
+
+```mjs
+import encoding from 'node:encoding';
+```
+
+```cjs
+const encoding = require('node:encoding');
+```
+
+This module is only available under the `node:` scheme. The following will not
+work:
+
+```mjs
+import encoding from 'encoding';
+```
+
+```cjs
+const encoding = require('encoding');
+```
+
+## `validateAscii(input)`
```

Comment:
Given that it returns a boolean, I'd call it something like `isValidAscii`.

Reply:
Good catch! I'm updating the pull request with `isAscii` and `isUtf8`